SharedCode/src/commonMain/kotlin/ApplicationPresenter.kt

```diff
@@ -19,18 +19,25 @@ class ApplicationPresenter: ApplicationContract.Presenter() {
     private var view: ApplicationContract.View? = null
     private val job: Job = SupervisorJob()
 
+    private val stationCodes = listOf("KGX", "WNS", "WKM", "GLD", "WOK")
+    private val stationNames = MutableList<String>(stationCodes.size) { _ -> "" }
```

Comment:
What was the reasoning behind initialising this to a specific size? I find this pattern of setting the list to be a certain size and then adding the stations to a specific point, rather than just adding them to the end, quite surprising

Reply:
Ah this is a relic of something I had earlier using List rather than MutableList - I intended to make a list of a specific size for memory purposes, but that's not really happening any more as I'm using a MutableList - I'll make sure to fix this